parsers/ENTSOE.py

```diff
@@ -1065,3 +1081,43 @@ def fetch_consumption_forecast(zone_key, session=None, target_datetime=None,
             })
 
         return data
+
+
+def fetch_wind_solar_forecasts(zone_key, session=None, target_datetime=None,
+                               logger=logging.getLogger(__name__)):
+    """
+    Gets values and corresponding datetimes for all production types in the
+    specified zone. Removes any values that are in the future or don't have
+    a datetime associated with them.
+    Returns a list of dictionaries that have been validated.
+    """
+    if not session:
+        session = requests.session()
+    domain = ENTSOE_DOMAIN_MAPPINGS[zone_key]
+    # Grab production
+    parsed = parse_production(
+        query_wind_solar_production_forecast(domain, session,
+                                             target_datetime=target_datetime))
+
+    if not parsed:
+        return None
+
+    productions, production_dates = parsed
```

Comment:
not a big deal, but I find it weird that apart from the method name, everything is named production as if it was real values. Maybe rename `productions` to `prod_forecasts`?